diff --git a/test/metabase/api/table_test.clj b/test/metabase/api/table_test.clj
index a348540ca2..c81ac9b109 100644
--- a/test/metabase/api/table_test.clj
+++ b/test/metabase/api/table_test.clj
@@ -706,7 +706,7 @@
                    (dimension-options-for-field response "timestamp"))))))
 
       (testing "time columns"
-        (mt/test-drivers (mt/normal-drivers-except #{:sparksql :mongo :oracle :redshift})
+        (mt/test-drivers (mt/normal-drivers-except #{:sparksql :mongo :oracle :exasol :redshift})
           (mt/dataset test-data-with-time
             (let [response (mt/user-http-request :rasta :get 200 (format "table/%d/query_metadata" (mt/id :users)))]
               (is (= []
diff --git a/test/metabase/driver/sql_jdbc_test.clj b/test/metabase/driver/sql_jdbc_test.clj
index 5e8c012138..b47b53622e 100644
--- a/test/metabase/driver/sql_jdbc_test.clj
+++ b/test/metabase/driver/sql_jdbc_test.clj
@@ -195,7 +195,7 @@
                  (spliced-count-of :checkins [:= $date "2014-03-05"]))))))
 
     ;; Oracle, Redshift, and SparkSQL don't have 'Time' types
-    (mt/test-drivers (disj (sql-jdbc.tu/sql-jdbc-drivers) :oracle :redshift :sparksql)
+    (mt/test-drivers (disj (sql-jdbc.tu/sql-jdbc-drivers) :oracle :exasol :redshift :sparksql)
       (testing "splicing a time"
         (is (= 2
                (mt/dataset test-data-with-time
diff --git a/test/metabase/query_processor/middleware/format_rows_test.clj b/test/metabase/query_processor/middleware/format_rows_test.clj
index 5608a9b374..044e42a2d2 100644
--- a/test/metabase/query_processor/middleware/format_rows_test.clj
+++ b/test/metabase/query_processor/middleware/format_rows_test.clj
@@ -14,7 +14,7 @@
 (def ^:private dbs-exempt-from-format-rows-tests
   "DBs to skip the tests below for. TIMEZONE FIXME — why are so many databases not running these tests? Most of these
   should be able to pass with a few tweaks."
-  #{:oracle :mongo :redshift :presto :sparksql :snowflake})
+  #{:oracle :exasol :mongo :redshift :presto :sparksql :snowflake})
 
 (deftest format-rows-test
   (mt/test-drivers (mt/normal-drivers-except dbs-exempt-from-format-rows-tests)
diff --git a/test/metabase/query_processor_test/alternative_date_test.clj b/test/metabase/query_processor_test/alternative_date_test.clj
index f21aca9a7c..b8ce90f1c1 100644
--- a/test/metabase/query_processor_test/alternative_date_test.clj
+++ b/test/metabase/query_processor_test/alternative_date_test.clj
@@ -174,7 +174,7 @@
 (deftest iso-8601-text-fields
   (testing "text fields with semantic_type :type/ISO8601DateTimeString"
     (testing "return as dates"
-      (mt/test-drivers (disj (sql-jdbc.tu/sql-jdbc-drivers) :sqlite :oracle :sparksql)
+      (mt/test-drivers (disj (sql-jdbc.tu/sql-jdbc-drivers) :sqlite :oracle :sparksql :exasol)
         (is (= [[1 "foo" #t "2004-10-19T10:23:54" #t "2004-10-19" #t "10:23:54"]
                 [2 "bar" #t "2008-10-19T10:23:54" #t "2008-10-19" #t "10:23:54"]
                 [3 "baz" #t "2012-10-19T10:23:54" #t "2012-10-19" #t "10:23:54"]]
@@ -203,6 +203,16 @@
                             (qp/process-query
                               (assoc (mt/mbql-query just-dates)
                                      :middleware {:format-rows? false}))))))))
+      (testing "exasol doesn't have a time type"
+        (mt/test-drivers #{:exasol}
+                         (is (= [[1 "foo" #t "2004-10-19T10:23:54" #t "2004-10-19"]
+                                 [2 "bar" #t "2008-10-19T10:23:54" #t "2008-10-19"]
+                                 [3 "baz" #t "2012-10-19T10:23:54" #t "2012-10-19"]]
+                 ;; string-times dataset has three text fields, ts, d, t for timestamp, date, and time
+                                (mt/rows (mt/dataset just-dates
+                                                     (qp/process-query
+                                                      (assoc (mt/mbql-query just-dates)
+                                                             :middleware {:format-rows? false}))))))))
       (testing "sqlite returns as strings"
         (mt/test-drivers #{:sqlite}
           (is (= [[1 "foo" "2004-10-19 10:23:54" "2004-10-19" "10:23:54"]
@@ -216,13 +226,13 @@
     (testing "are queryable as dates"
       (testing "a datetime field"
         ;; TODO: why does this fail on oracle? gives a NPE
-        (mt/test-drivers (disj (sql-jdbc.tu/sql-jdbc-drivers) :oracle :sparksql)
+        (mt/test-drivers (disj (sql-jdbc.tu/sql-jdbc-drivers) :oracle :sparksql :exasol)
           (is (= 1
                  (count (mt/rows (mt/dataset string-times
                                    (mt/run-mbql-query times
                                      {:filter   [:= [:datetime-field $ts :day] "2008-10-19"]}))))))))
       (testing "a date field"
-        (mt/test-drivers (disj (sql-jdbc.tu/sql-jdbc-drivers) :oracle :sparksql)
+        (mt/test-drivers (disj (sql-jdbc.tu/sql-jdbc-drivers) :oracle :sparksql :exasol)
           (is (= 1
                  (count (mt/rows (mt/dataset string-times
                                    (mt/run-mbql-query times
diff --git a/test/metabase/query_processor_test/date_bucketing_test.clj b/test/metabase/query_processor_test/date_bucketing_test.clj
index 116f05dba3..be843509a0 100644
--- a/test/metabase/query_processor_test/date_bucketing_test.clj
+++ b/test/metabase/query_processor_test/date_bucketing_test.clj
@@ -272,7 +272,7 @@
   ;; timezone
   ;;
   ;; TIMEZONE FIXME
-  (mt/test-drivers (mt/normal-drivers-except #{:h2 :sqlserver :redshift :sparksql :mongo})
+  (mt/test-drivers (mt/normal-drivers-except #{:h2 :sqlserver :redshift :sparksql :mongo :exasol}) ; https://github.com/exasol/metabase-driver/issues/9
     (testing "Change JVM timezone from UTC to Pacific"
       (is (= (cond
                (= :sqlite driver/*driver*)
@@ -524,7 +524,7 @@
     ;; timezone
     ;;
     ;; TIMEZONE FIXME
-    (mt/test-drivers (mt/normal-drivers-except #{:h2 :sqlserver :redshift :sparksql :mongo :vertica})
+    (mt/test-drivers (mt/normal-drivers-except #{:h2 :sqlserver :redshift :sparksql :mongo :vertica :exasol}) ; https://github.com/exasol/metabase-driver/issues/9
       (is (= (cond
                (= :sqlite driver/*driver*)
                (results-by-day u.date/parse date-without-time-format-fn [6 10 4 9 9 8 8 9 7 9])
@@ -694,7 +694,7 @@
   ;; timezone (TIMEZONE FIXME)
   (testing "JVM timezone set to Pacific"
     (mt/test-drivers (mt/normal-drivers-except #{:h2 :sqlserver :redshift :sparksql :mongo :bigquery
-                                                 :bigquery-cloud-sdk})
+                                                 :bigquery-cloud-sdk :exasol}) ; https://github.com/exasol/metabase-driver/issues/9
       (is (= (cond
                (= :sqlite driver/*driver*)
                (results-by-week u.date/parse
diff --git a/test/metabase/query_processor_test/parameters_test.clj b/test/metabase/query_processor_test/parameters_test.clj
index f1de41d12e..4d1adbb3f2 100644
--- a/test/metabase/query_processor_test/parameters_test.clj
+++ b/test/metabase/query_processor_test/parameters_test.clj
@@ -114,7 +114,7 @@
         ;; TIMEZONE FIXME — The excluded drivers below don't have TIME types, so the `attempted-murders` dataset doesn't
         ;; currently work. We should use the closest equivalent types (e.g. `DATETIME` or `TIMESTAMP` so we can still
         ;; load the dataset and run tests using this dataset such as these, which doesn't even use the TIME type.
-        (when-not (#{:oracle :presto :redshift :sparksql :snowflake} driver/*driver*)
+        (when-not (#{:oracle :exasol :presto :redshift :sparksql :snowflake} driver/*driver*)
           (mt/dataset attempted-murders
             (doseq [field
                     [:datetime
diff --git a/test/metabase/query_processor_test/remapping_test.clj b/test/metabase/query_processor_test/remapping_test.clj
index 70b172130f..64d4df2e53 100644
--- a/test/metabase/query_processor_test/remapping_test.clj
+++ b/test/metabase/query_processor_test/remapping_test.clj
@@ -170,7 +170,7 @@
   ;;
   ;; Having a self-referencing FK is currently broken with the Redshift and Oracle backends. The issue related to fix
   ;; this is https://github.com/metabase/metabase/issues/8510
-  (mt/test-drivers (disj (mt/normal-drivers-with-feature :foreign-keys) :redshift :oracle :vertica)
+  (mt/test-drivers (disj (mt/normal-drivers-with-feature :foreign-keys) :redshift :oracle :vertica :exasol) ; Fails with an exception, related to https://github.com/metabase/metabase/issues/18580
     (mt/dataset test-data-self-referencing-user
       (mt/with-column-remappings [users.created_by users.name]
         (db/update! Field (mt/id :users :created_by)
diff --git a/test/metabase/query_processor_test/time_field_test.clj b/test/metabase/query_processor_test/time_field_test.clj
index e8b2743998..fdfc7e6e28 100644
--- a/test/metabase/query_processor_test/time_field_test.clj
+++ b/test/metabase/query_processor_test/time_field_test.clj
@@ -14,7 +14,7 @@
 
 ;; TIMEZONE FIXME
 (def ^:private skip-time-test-drivers
-  #{:oracle :mongo :redshift :sparksql})
+  #{:oracle :exasol :mongo :redshift :sparksql})
 
 (deftest basic-test
   (mt/test-drivers (mt/normal-drivers-except skip-time-test-drivers)
diff --git a/test/metabase/query_processor_test/timezones_test.clj b/test/metabase/query_processor_test/timezones_test.clj
index 0eb62eed08..3312cb3d79 100644
--- a/test/metabase/query_processor_test/timezones_test.clj
+++ b/test/metabase/query_processor_test/timezones_test.clj
@@ -18,6 +18,7 @@
   https://circleci.com/workflow-run/856f6dd0-3d95-4732-a56e-1af59e3ae4ba. The goal is to gradually remove these
   one-by-one as they are fixed."
   #{:oracle
+    :exasol
     :presto
     :redshift
     :snowflake
