diff --git a/test/metabase/driver/sql/parameters/substitute_test.clj b/test/metabase/driver/sql/parameters/substitute_test.clj
index 58121dda9d..0c800f316b 100644
--- a/test/metabase/driver/sql/parameters/substitute_test.clj
+++ b/test/metabase/driver/sql/parameters/substitute_test.clj
@@ -993,7 +993,7 @@
 
 (deftest e2e-parse-native-dates-test
   (testing "Native dates should be parsed with the report timezone"
-    (mt/test-drivers (disj (sql-parameters-engines) :sqlite)
+    (mt/test-drivers (disj (sql-parameters-engines) :sqlite :exasol)
       (mt/with-report-timezone-id! "America/Los_Angeles"
         (let [query {:database   (mt/id)
                      :type       :native
diff --git a/test/metabase/query_processor/middleware/format_rows_test.clj b/test/metabase/query_processor/middleware/format_rows_test.clj
index d363fd2526..257a3fac95 100644
--- a/test/metabase/query_processor/middleware/format_rows_test.clj
+++ b/test/metabase/query_processor/middleware/format_rows_test.clj
@@ -19,7 +19,7 @@
   should be able to pass with a few tweaks. Some of them are excluded because they do not have a TIME data type and
   can't load the `time-test-data` dataset; but that's not true of ALL of these. Please make sure you add a note
   as to why a certain database is explicitly skipped if you skip it -- Cam"
-  #{:bigquery-cloud-sdk :oracle :mongo :redshift :sparksql :snowflake})
+  #{:bigquery-cloud-sdk :oracle :mongo :redshift :sparksql :snowflake :exasol}) ; Exasol does not support TIME data type
 
 (deftest format-rows-test
   (mt/test-drivers (filter mt/supports-time-type? (mt/normal-drivers-except dbs-exempt-from-format-rows-tests))
diff --git a/test/metabase/query_processor_test/alternative_date_test.clj b/test/metabase/query_processor_test/alternative_date_test.clj
index 8ebcdf186f..fea9f9bd59 100644
--- a/test/metabase/query_processor_test/alternative_date_test.clj
+++ b/test/metabase/query_processor_test/alternative_date_test.clj
@@ -51,7 +51,7 @@
                                            :query    {:source-table (str "card__" card-id)}}))))))))
 
 (deftest ^:parallel microseconds-test
-  (mt/test-drivers (disj (mt/normal-drivers) :sqlite)
+  (mt/test-drivers (disj (mt/normal-drivers) :sqlite :exasol) ; Timestamp/timezone covered by Exasol specific tests.
     (let [results (get {:sqlite #{[1 4 "2015-06-06 10:40:00"] [2 0 "2015-06-10 19:51:00"]}
                         :oracle #{[1M 4M "2015-06-06T10:40:00Z"] [2M 0M "2015-06-10T19:51:00Z"]}}
                        driver/*driver*
@@ -200,7 +200,7 @@
     (testing "return as dates"
       (mt/test-drivers (-> (sql-jdbc.tu/normal-sql-jdbc-drivers)
                            (conj :bigquery-cloud-sdk)
-                           (disj :sqlite :oracle :sparksql))
+                           (disj :sqlite :oracle :sparksql :exasol)) ; Exasol does not support TIME data type
         (is (= [[1 "foo" #t "2004-10-19T10:23:54" #t "2004-10-19" #t "10:23:54"]
                 [2 "bar" #t "2008-10-19T10:23:54" #t "2008-10-19" #t "10:23:54"]
                 [3 "baz" #t "2012-10-19T10:23:54" #t "2012-10-19" #t "10:23:54"]]
@@ -267,7 +267,7 @@
       (mt/dataset string-times
         (testing "a datetime field"
           ;; TODO: why does this fail on oracle? gives a NPE
-          (mt/test-drivers (disj (sql-jdbc.tu/normal-sql-jdbc-drivers) :oracle :sparksql)
+          (mt/test-drivers (disj (sql-jdbc.tu/normal-sql-jdbc-drivers) :oracle :sparksql :exasol) ; Exasol does not support TIME data type
             (is (= 1
                    (->> (mt/run-mbql-query times
                           {:filter [:= !day.ts "2008-10-19"]})
@@ -282,7 +282,7 @@
                         mt/rows
                         count)))))
         (testing "a date field"
-          (mt/test-drivers (disj (sql-jdbc.tu/normal-sql-jdbc-drivers) :oracle :sparksql)
+          (mt/test-drivers (disj (sql-jdbc.tu/normal-sql-jdbc-drivers) :oracle :sparksql :exasol) ; Exasol does not support TIME data type
             (is (= 1
                    (->> (mt/run-mbql-query times
                           {:filter [:= !day.d "2008-10-19"]})
diff --git a/test/metabase/query_processor_test/explicit_joins_test.clj b/test/metabase/query_processor_test/explicit_joins_test.clj
index 4e27fd2ae6..7f27236a4b 100644
--- a/test/metabase/query_processor_test/explicit_joins_test.clj
+++ b/test/metabase/query_processor_test/explicit_joins_test.clj
@@ -860,7 +860,7 @@
                                         results))))))))))
 
 (deftest ^:parallel double-quotes-in-join-alias-test
-  (mt/test-drivers (mt/normal-drivers-with-feature :left-join)
+  (mt/test-drivers (disj (mt/normal-drivers-with-feature :left-join) :exasol) ; Exasol does not support '.' in quoted identifiers: "Error executing query: '.' character is not allowed within quoted identifiers"
     (testing "Make sure our we handle (escape) double quotes in join aliases. Make sure we prevent SQL injection (#20307)"
       (let [expected-rows (mt/rows
                            (mt/run-mbql-query venues
diff --git a/test/metabase/query_processor_test/parameters_test.clj b/test/metabase/query_processor_test/parameters_test.clj
index 81cad26525..4e2e9a8f5e 100644
--- a/test/metabase/query_processor_test/parameters_test.clj
+++ b/test/metabase/query_processor_test/parameters_test.clj
@@ -348,6 +348,7 @@
                         ;; These do not support ParameterMetadata.getParameterCount
                         (remove #{:athena
                                   :bigquery-cloud-sdk
+                                  :exasol ; Fails with exception "Error preparing statement: object VENUES not found [line 4, column 2] (Session: 1779010379713609728) {:driver :exasol, :type :driver, :sql ["-- Metabase" "SELECT" "  *" "FROM" "  -- ?" "  VENUES" "WHERE" "  ? = price"], :params ["foobar" "foobar"]}"
                                   :presto-jdbc
                                   :redshift
                                   :snowflake
diff --git a/test/metabase/query_processor_test/timezones_test.clj b/test/metabase/query_processor_test/timezones_test.clj
index 1b3c39c1a4..1ac0587913 100644
--- a/test/metabase/query_processor_test/timezones_test.clj
+++ b/test/metabase/query_processor_test/timezones_test.clj
@@ -24,6 +24,7 @@
   #{:athena
     :bigquery-cloud-sdk
     :oracle
+    :exasol ; Timestamp/timezone covered by Exasol specific tests.
     :redshift
     :sparksql
     :vertica})
diff --git a/test/metabase/test/initialize.clj b/test/metabase/test/initialize.clj
index f9b32da937..0cb97eda18 100644
--- a/test/metabase/test/initialize.clj
+++ b/test/metabase/test/initialize.clj
@@ -22,7 +22,7 @@
                                      (str/join "\n" [border body border])
                                      "\n")))))
 
-(def ^:private init-timeout-ms (u/seconds->ms 60))
+(def ^:private init-timeout-ms (u/seconds->ms 120)) ; Integration tests fail on GitHub Action with a timeout
 
 (def ^:private ^:dynamic *initializing*
   "Collection of components that are being currently initialized by the current thread."
