diff --git a/modules/drivers/athena/deps.edn b/modules/drivers/athena/deps.edn
index 1c138dcbc1..f864c0f933 100644
--- a/modules/drivers/athena/deps.edn
+++ b/modules/drivers/athena/deps.edn
@@ -1,9 +1,7 @@
 {:paths
  ["src" "resources"]
 
- :mvn/repos
- {"athena" {:url "https://s3.amazonaws.com/maven-athena"}}
-
  :deps
- {com.amazonaws/aws-java-sdk-core {:mvn/version "1.12.392"}
-  com.metabase/athena-jdbc {:mvn/version "2.0.35"}}}
+ {com.amazonaws/aws-java-sdk-core {:mvn/version "1.12.392"}}
+ ; Download of Athena JDBC driver from S3 fails in GitHub Action and it is not required for building.
+}
