diff --git a/modules/drivers/athena/deps.edn b/modules/drivers/athena/deps.edn
index 5a5f08f1d9..0250bbd925 100644
--- a/modules/drivers/athena/deps.edn
+++ b/modules/drivers/athena/deps.edn
@@ -1,8 +1,5 @@
 {:paths
  ["src" "resources"]
 
- :mvn/repos
- {"athena" {:url "https://s3.amazonaws.com/maven-athena"}}
-
- :deps
- {com.metabase/athena-jdbc {:mvn/version "2.0.33"}}}
+; Download of Athena JDBC driver from S3 fails in GitHub Action and it is not required for building.
+}
+
